core/sockets.py

```diff
@@ -65,16 +50,15 @@ def process_from_socket(self, context):
 
 class SvSocketCommon:
     """ Base class for all Sockets """
+    color = (1, 0, 0, 1)  # base color, other sockets should override the property, use FloatProperty for dynamic
+    quick_link_to_node = str()  # sockets which often used with other nodes can fill its `bl_idname` here
+
     use_prop: BoolProperty(default=False)
 
-    use_expander: BoolProperty(default=True)
-    use_quicklink: BoolProperty(default=True)
     expanded: BoolProperty(default=False)
     custom_draw: StringProperty(description="For name of method which will draw socket UI (optionally)")
     prop_name: StringProperty(default='', description="For displaying node property in socket UI")
 
-    quicklink_func_name: StringProperty(default="", name="quicklink_func_name")
```

Comment:
ok.. but this was added because it was useful to allow this operator to spawn not just one Node + attach, but spawn several as a product of some function defined in the node itself. It was intentionally broad.

Reply:
We can bring it back any moment but in current branch no one node use this method.

It is possible to write next node now.

```Python
def sv_init(self):
    sock = self.inputs.new('SvVerticesSocket', 'Verts')
    sock.quick_link_to_node = 'bl_idname_of_node_which_should_be_linked'
```

And the socket of particular node instance will have custom quick link.